### ISSUE TITLE
Where are the files?? HORROR TRIP

### ISSUE THREAD

**Issue by DominikVanyi** [Dimowner/AudioRecorder#95]
I made several recordings and I am unable to find them on my device. It says file location is - after I set to Store Records in a public dir -  /storage/emulated... .. 

I do not have any emulated folder on my phone... 
How on Earth can I find the files which the app created so that I can download them to my computer...

**Comment by firefoxlover** [comment]
probably in the apps storage

**Comment by Dimowner** [comment]
All recorded files are stored in app's private directory which is not available for user (unless you root permission).
If you want to get record file you can use 'Share' or 'Download' feature. With 'share' you can send selected record anywhere you want, for example upload to your Google Disc. With 'Download' you can copy selected record into Downloads directory which is available for user.

**Comment by cyberhardt** [comment]
I can access that folder only with the default file manager. for all other file managers, (...)/Android/data is not acessible (unless you have root)

**Comment by firefoxlover** [comment]
yes agree, it would be great if the app was able to opt-in use storage access framework and save to a specified directory.

**Comment by sumade123** [comment]
Yeah that's a real shame, I love the app but can't really use it now as can't do auto backup of files - does anyone know how hard it would be to change/allow selection of the default storage location? Thanks

**Comment by jayb-g** [comment]
@Dimowner this is definitely much needed. Especially when you change devices/format you want to take all the recordings to your new phone. Although the app let's you 'download' the recordings currently, there's no easy way to _restore_ them by just copying to the app's private directory. 

Built-in backup and restore feature would be perfect for that. Encrypted backups would be even better.
And meanwhile maybe also allow to let adb backup your app data until the backup/restore is implemented?

**Comment by AlexandreBonneau** [comment]
Maybe this can be closed, since this request has been done on #112, #105, #95, #76 and originally in #67.
#37 is linked to it as well, but was more asking about the ability to create multiple folders.
